edx_sga/tests.py

```diff
@@ -129,22 +151,34 @@ def make_student(self, block, name, make_state=True, **state):
         }
 
     def personalize(self, block, module, item, submission):
+        """
+        Set values on block from student state.
+        """
```

Comment:
Go ahead and add: `# pylint: disable=unused-argument` here since submission is in fact used.